hooks/queries/votes/useAugmentedVoteData.ts

```diff
@@ -1,27 +1,32 @@
+import assert from "assert";
 import { useQuery } from "@tanstack/react-query";
 import { augmentedVoteDataKey } from "constant";
 import { getAugmentedVoteData } from "web3";
-import { useActiveVotes } from "./useActiveVotes";
-import { usePastVotes } from "./usePastVotes";
-import { useUpcomingVotes } from "./useUpcomingVotes";
 
-export function useAugmentedVoteData() {
-  const { data: activeVotes } = useActiveVotes();
-  const { data: upcomingVotes } = useUpcomingVotes();
-  const { data: pastVotes } = usePastVotes();
-
-  const allVotes = { ...activeVotes, ...upcomingVotes, ...pastVotes };
-
-  const queryResult = useQuery({
+export function useAugmentedVoteData(
+  params: Partial<{
+    ancillaryData: string;
+    time: number;
+    identifier: string;
+  }>
```

Comment:
previously when calling this, it would fetch all augmented data for all votes.  now we pass params to specify the single vote we care about